src/cephadm/cephadm

```diff
@@ -1566,7 +1588,11 @@ def command_run():
     c = get_container(args.fsid, daemon_type, daemon_id)
     command = c.run_cmd()
     logger.debug("Running command: %s" % ' '.join(command))
-    return subprocess.call(command)
+    try:
+        return subprocess.call(command, timeout=args.timeout) # py3
+    except TypeError:
+        # TODO: py2 has no timeout arg
```

Comment:
There are four copies of this code now. We already have a wrapper for `subprocess.call`. can we simply use this here? Alternatively we need another wrapper.

Reply:
consolidated into a `call_timeout` function: https://github.com/ceph/ceph/commit/b24b2bfa5bcc7428b9392a6dfc2fabb1f334dc52#diff-8b586ec9c3ad3e8421a8858888f7ddf0R255-R291